src/applications/vaos/actions/expressCare.js

```diff
@@ -168,72 +193,77 @@ export function routeToPreviousAppointmentPage(router, current) {
   );
 }
 
-async function buildPreferencesDataAndUpdate(expressCare) {
+async function buildPreferencesDataAndUpdate(data) {
   const preferenceData = await getPreferences();
-  const preferenceBody = createPreferenceBody(preferenceData, expressCare.data);
+  const preferenceBody = createPreferenceBody(preferenceData, data);
   return updatePreferences(preferenceBody);
 }
 
 export function submitExpressCareRequest(router) {
   return async (dispatch, getState) => {
     const expressCare = getState().expressCare;
-    const activeFacility = selectActiveExpressCareFacility(
-      getState(),
-      moment.utc(),
-    );
+    const formData = expressCare.newRequest.data;
+    const { reasonForRequest, phoneNumber, email } = formData;
 
-    dispatch({
-      type: FORM_SUBMIT,
-    });
+    if (!!reasonForRequest?.reason && !!phoneNumber && !!email) {
```

Comment:
For some reason, form validation is not being triggered before this `submitExpressCareRequest()` action.  Added this extra data check so that the submission doesn't run, but validation does.  I'm pretty sure this isn't the proper way to do it, but I couldn't figure out how to prevent submission if the form isn't valid.

Reply:
I made a comment below, I think we just need to remove the `onSubmit` prop from `FormButtons`. And maybe think of a way of making that less confusing.